src/library/vtkF3DRenderPass.cxx

```diff
@@ -229,13 +229,15 @@ void vtkF3DRenderPass::Blend(const vtkRenderState* s)
               "#define GOLDEN_ANGLE 2.399963\n"
               "vec3 BokehBlur(vec2 tcoords, float radius)"
               "{\n"
+              "  ivec2 textureSize = textureSize(texBackground, 0);\n"
+              "  float ratio = float(textureSize.x) / float(textureSize.y);\n"
               "  vec3 acc = vec3(0.0);\n"
               "  float factor = radius / sqrt(float(SAMPLES));\n"
               "  for (int i = 0; i < SAMPLES; i++)\n"
               "  {\n"
               "    float theta = float(i) * GOLDEN_ANGLE;\n"
               "    float r = factor * sqrt(float(i));\n"
-              "    vec2 pt = vec2(r * cos(theta), r * sin(theta));\n"
+              "    vec2 pt = vec2(r * cos(theta) / ratio, r * sin(theta));\n"
```

Comment:
ratio is not taken into account with y coord ?

Reply:
No it shouldn't